pkg/util/shell/shell.go

```diff
@@ -90,7 +90,7 @@ func GetPrefixSuffixDelimiterForSet(userShell string, pathVar bool) (prefix, suf
 		prefix = "SET "
 		suffix = "\n"
 		if pathVar {
-			suffix = ";%PATH%" + suffix
+			suffix = ";%PATH%;" + suffix
```

Comment:
it was a workaround, but seemed to be working, but I believe no-one tested this from code, but only from the command line itself. it should have no real effect. because the issue is with the way the environment variable gets evaluated.

(I suggested it as a quick test, and I believe @agajdosi ran it from `cmd` to verify)